packages/core/styles/components/_badge.scss

```
@@ -5,8 +5,8 @@ span[role='status'][data-intent] {
   font-size: var(--ix-size-sm);
   line-height: var(--ix-line-height);
 
-  padding: 0 var(--ix-space-sm);
-  border-radius: var(--ix-radius);
+  padding: 0 var(--ix-space-md);
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)